packages/insomnia-xpath/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.build.json",
+  "compilerOptions": {
+    "rootDir": "."
+  },
+  "include": [
+    "src",
+    "jest.config.js",
```

Comment:
🧹

Reply:
a decision needs to be made here https://kongstrong.slack.com/archives/GPARXT60G/p1616428970018300

Reply:
in the name of expediency I went through every tsconfig and made sure none had trailing commas.  turns out there were 3 others that had one somewhere.
![Screenshot_20210324_094937](https://user-images.githubusercontent.com/15232461/112321296-3fdebb80-8c86-11eb-9afd-0ea7a00cf85b.png)

I fixed them all